### PR TITLE
pypy: bootstrap with pypy-6.0.0

### PR DESCRIPTION
**Pull request by maxim-belkin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Local (successful) installation: https://gist.github.com/edc0808ed98c342e4dfeb573dc9eba36 (apparently, this one was done using another version of pypy, presumably 5.1.1)

### CODE REVIEW

**Comment by DomT4** [comment]
Isn't this the release that has silly linkage in the binary that won't work if `HOMEBREW_PREFIX` isn't `/usr/local`? 🤔

**Comment by maxim-belkin** [comment]
Not sure about libressl. 🤔 

Re failure: I don't understand what causes `sha256` mismatch.

**Comment by DomT4** [comment]
I _think_ LibreSSL landed in El Capitan, IIRC. I get the same `sha256` as the CI FWIW:
```
d7dc443e6bb9a45212e8d8f5a63e9f6ce23f1d88c50709efea1c75b76c8bc186  pypy2-v6.0.0-osx64.tar.bz2
```

**Comment by maxim-belkin** [comment]
I see what happened. I tried a different version of bootstrapping `pypy` before trying `6.0.0`, and when I switched to `6.0.0` Homebrew said that it already has that resource and did not download it, so reported me the `sha256` for the other pypy version

**Comment by maxim-belkin** [comment]
That, in part, happened because Homebrew renames downloaded resources.

File: Formula/pypy.rb [hunk 1]
```diff
@@ -24,8 +24,8 @@ class Pypy < Formula
   depends_on "openssl"
 
   resource "bootstrap" do
-    url "https://bitbucket.org/pypy/pypy/downloads/pypy-2.5.0-osx64.tar.bz2"
-    sha256 "30b392b969b54cde281b07f5c10865a7f2e11a229c46b8af384ca1d3fe8d4e6e"
+    url "https://bitbucket.org/pypy/pypy/downloads/pypy2-v6.0.0-osx64.tar.bz2"
```
Comment (ilovezfs) [inline Formula/pypy.rb]: should set an explicit version here
```
version "6.0.0"
```

**Comment by ilovezfs** [comment]
Thanks @maxim-belkin! Shipped.

**Pull request closed without merging**